proto/p4/p4runtime.proto

```diff
@@ -136,6 +136,14 @@ message TableEntry {
   // this is stored, as long as it is returned with the rest of the entry in
   // a Read RPC.
   uint64 controller_metadata = 5;
+  // meter_config and counter_data are convenience fields that enable the
+  // controller to configure the direct resources associated with the table at
+  // the same time as a match-action entry is inserted or modified. When
```

Comment:
I think it is up to the architecture (in this case PSA) to define default behavior for "uninitialized" resources. In the meter case, I seem to recall that the behavior would be to tag all packets as GREEN. Now I guess the question is what would be returned by the Read RPC in that case...